src/plugins/manualRowResize/test/manualRowResize.e2e.js

```diff
@@ -597,8 +597,8 @@ describe('manualRowResize', () => {
       expect($rowHeader.offset().top + $rowHeader.height() - 5).toBeCloseTo($handle.offset().top, 0);
       expect($rowHeader.offset().left).toBeCloseTo($handle.offset().left, 0);
 
+      $(mainHolder).scrollTop(1); // we have to trigger innerBorderTop before we scroll to correct position
```

Comment:
On my environment macOS + FF81 the test still doesn't pass 🤔 
![Zrzut ekranu 2020-09-28 o 11 52 12](https://user-images.githubusercontent.com/571316/94418914-50ecbb00-0182-11eb-88dc-8b66fd82069a.png)

However, after giving the HoT time to reacting to the scroll event it seems that it works. 
```suggestion
      $(mainHolder).scrollTop(1); // we have to trigger innerBorderTop before we scroll to correct position
      await sleep(10);
```

Reply:
Thanks, maybe we can set in advance 100ms?

Reply:
Sure 👍